homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -120,3 +153,24 @@ class AutomowerControlEntity(AutomowerAvailableEntity):
     def available(self) -> bool:
         """Return True if the device is available."""
         return super().available and _check_error_free(self.mower_attributes)
+
+
+class WorkAreaControlEntity(AutomowerControlEntity):
+    """Replies available when the mower is connected."""
```

Comment:
stale docstring

Reply:
done in https://github.com/home-assistant/core/pull/126376/commits/7511f4dcfe2a815c2dfc42d8509c9853d310112c